doc/tntcxx_api.rst

```diff
@@ -28,141 +26,184 @@ Methods:
 
 .. _tntcxx_api_connector_connect:
 
-.. c:function:: int connect(Connection<BUFFER, NetProvider>& conn, const std::string_view& addr, unsigned port)
+..  c:function:: int connect(Connection<BUFFER, NetProvider>& conn, const std::string_view& addr, unsigned port)
 
-   Connects to a Tarantool instance that is listening on ``addr:port``.
+    Connects to a Tarantool instance. If a host doesn't reply within a timeout
+    period (2 seconds by default), an error is returned.
 
-   :param conn: connection object <cross-ref to corresp. topic>.
-   :param addr: URL of a host where a Tarantool instance is running.
-   :param port: a port that a Tarantool instance is listening to.
+    :param conn: connection object of the ``Connection`` class.
+    :param addr: address of the host where a Tarantool instance is running.
+    :param port: port that a Tarantool instance is listening to.
 
-   :return: connection status code.
-   :rtype: <TBD>
+    :return: ``0`` on success.
+    :return: ``-1`` otherwise. An error message is set that can be get by
+             the ``connection.getError()`` method.
+    :rtype: int
 
-   **Possible errors:** <TBD>
+    **Possible errors:**
 
-   **Complexity factors:** <TBD>
+    *   connection timeout
+    *   refused to connect (incorrect address or/and port)
+    *   system errors. For example, a socket cannot be created or any of the
```

Comment:
cannot → can not | can't

Reply:
`can not` is not the case (https://www.grammarly.com/blog/cannot-or-can-not/)
If we want to be less formal and use contractions as the primary option, let's have this as a guideline (Microsoft does: https://docs.microsoft.com/en-gb/style-guide/word-choice/use-contractions)). I can write the guideline.

Reply:
Corrected.